proto/ai/traceable/agent/config/v1/config.proto

```diff
@@ -30,6 +30,8 @@ message Opa {
   // Certificate filename containing the CA to verify the server's certificate.
   // If this is non-empty, you shoulds `https` for the protocol in `endpoint` above. 
   google.protobuf.StringValue cert_file = 4;
+  // set this flag to use https connection when the provided certificate path is empty
+  google.protobuf.BoolValue use_secure_connection = 5;
```

Comment:
can you run `make generate`?

Reply:
See https://github.com/Traceableai/agent-config#making-changes-to-configproto

Reply:
I didnt know this. Updated the pr